### PR TITLE
Add create & update timestamps to all models

### PR DESCRIPTION
**Pull request by monikasulik**
https://uktrade.atlassian.net/browse/ED-499

### CODE REVIEW

File: user/models.py [hunk 1]
```diff
@@ -24,7 +24,7 @@ class User(models.Model):
         unique=True,
     )
     company = models.ForeignKey(
-        Company, related_name='users', null=False
+        Company, related_name='users', null=True
```
Comment (monikasulik) [inline user/models.py]: Reason for change: https://uktrade.atlassian.net/browse/ED-512

Comment (richtier) [inline user/models.py]: I take it we're setting `null` to `True` so those 15 tests pass?

Comment (konradko) [inline user/models.py]: Incorrect / broken tests are not a good reason to model the data incorrectly. Users without a company are not allowed in the system, which has to be reflected on the DB level. We can do that in a separate ticket, given it used to be nullable and there were no migrations for it.

Comment (monikasulik) [inline user/models.py]: I already created a separate ticket to do this - it's at the start of this thread. Feel free to schedule it into this sprint if you feel it's urgent.

File: company/migrations/0015_auto_20161129_1621.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
```
Comment (konradko) [inline company/migrations/0015_auto_20161129_1621.py]: looks like a conflicting migration - some fields were added earlier (probably you just have to run migrate before generating the migration)

Comment (monikasulik) [inline company/migrations/0015_auto_20161129_1621.py]: Should be fixed now

**Pull request merged**